app/Providers/PhoenixServiceProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rogue\Providers;
+
+use Rogue\Services\Phoenix\Phoenix;
+use Illuminate\Support\ServiceProvider;
+
+class PhoenixServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap the application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        //
+    }
+
+    /**
+     * Register the application services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->app->singleton('phoenix', function ($app) {
```

Comment:
It shouldn't be necessary to make this class into a singleton (since it doesn't have to maintain any state between calls made during a single request), so you can probably remove this service provider & let the service container handle finding and instantiating the class.